mesos_slave/README.md

```diff
@@ -48,18 +48,17 @@ If you are not using DC/OS, then use either the Marathon web UI or post to the A
       {"containerPath": "/host/sys/fs/cgroup","hostPath": "/sys/fs/cgroup","mode": "RO"}
     ],
     "docker": {
-      "image": "datadog/docker-dd-agent:11.0.5160",
+      "image": "datadog/agent:latest",
       "network": "BRIDGE",
       "portMappings": [
         {"containerPort": 8125,"hostPort": 8125,"servicePort": 10000,"protocol": "udp","labels": {}},
         {"containerPort": 9001,"hostPort": 9001,"servicePort": 10001,"protocol": "tcp","labels": {}}
       ],
       "privileged": false,
       "parameters": [
-        {"key": "name","value": "dd-agent"},
-        {"key": "env","value": "API_KEY=DATADOGAPIKEY"},
-        {"key": "env","value": "MESOS_SLAVE=true"},
-        {"key": "env","value": "SD_BACKEND=docker"}
+        {"key": "name","value": "datadog-agent"},
+        {"key": "env","value": "DD_API_KEY=DATADOGAPIKEY"},
```

Comment:
According to the contributing Guidelines we shoud use `<DATADOG_API_KEY>` instead of `DATADOGAPIKEY`